webpack/assets/javascripts/react_app/components/AutoCompleteSearch/AutoCompleteReducer.js

```diff
@@ -0,0 +1,41 @@
+import Immutable from 'seamless-immutable';
+import { STATUS } from '../../constants';
+import {
+  AUTO_COMPLETION_REQUEST,
+  AUTO_COMPLETION_SUCCESS,
+  AUTO_COMPLETION_FAILURE,
+} from './AutoCompleteConstants';
+
+const initialState = Immutable({
+  options: [],
+  status: 'PENDING',
+  error: null,
+  searchQuery: '',
```

Comment:
do we care about the original search query vs the current search query? (assuming one starts typing and change the existing filter prior to clicking on search)